File: Plugins/Aspose_Slides_Java_for_PHP/src/aspose/slides/WorkingWithSlidesInPresentation/ChangingPosition.php
<?php

namespace Aspose\Slides\WorkingWithSlidesInPresentation;

use com\aspose\slides\Presentation as Presentation;
use com\aspose\slides\SaveFormat as SaveFormat;

class ChangingPosition {

    public static function run($dataDir=null)
    {

        # Instantiate Presentation class that represents the presentation file
        $pres = new Presentation($dataDir . 'Aspose.pptx');

        # Get the slide whose position is to be changed
        $slide = $pres->getSlides()->get_Item(0);

        # Set the new position for the slide
        $slide->setSlideNumber(2);

        # Saving the presentation
        $save_format = new SaveFormat();
        $pres->save($dataDir . "Aspose_Position.pptx", $save_format->Pptx);

        print "Changes slide position, please check the output file." . PHP_EOL;

    }


}

File: Plugins/Aspose-Slides-Java-for-Jython/asposeslides/WorkingWithSlidesInPresentation/ChangingPosition.py
from asposeslides import Settings
from com.aspose.slides import Presentation
from com.aspose.slides import SaveFormat

class ChangingPosition:

    def __init__(self):
        
        dataDir = Settings.dataDir + 'WorkingWithSlidesInPresentation/ChangingPosition/'
        
       # Instantiate Presentation class that represents the presentation file
        pres = Presentation(dataDir + 'Aspose.pptx')

        # Get the slide whose position is to be changed
        slide = pres.getSlides().get_Item(0)

        # Set the position for the slide
        slide.setSlideNumber(2)

        # Saving the presentation
        save_format = SaveFormat
        pres.save(dataDir + "Aspose_Position.pptx", save_format.Pptx)

        print "Changes slide position, please check the output file."

if __name__ == '__main__':        
    ChangingPosition()